lib/stat_tracker.rb

```diff
@@ -11,4 +13,16 @@ def initialize(data)
     @data = data
   end
 
+  def highest_total_score
+    games_table = CSV.parse(File.read(self.data), headers: true).by_row
```

Comment:
I'm not sure if you're having problems with the tests taking too long, but if you are, you might consider putting your CSV.parse methods in the initialize. I did that and my tests don't take longer than 2-3sec to run through.